oaknational/Oak-Web-Application
# fix: enable local development in chrome [LESQ-1052]

## Description

Music year: 1983

[Ticket](https://www.notion.so/oaknationalacademy/I-want-to-develop-locally-on-Chrome-d26e3f1f31c94c5c8b89605f3f86e584?pvs=4)
- Adds reverse proxying to posthog calls when developing locally - this gets around the cors block that happens otherwise on localhost, preventing interaction with feature flags (see https://posthog.com/docs/advanced/proxy/nextjs) 
- Will only add this on dev mode, production builds should still use the current posthog set up

## Issue(s)

Fixes #
Unable to develop feature flagged features locally in chrome

## How to test

This should only affect local development, it should not affect how events are reported to posthog or how feature flags work

1. Go to https://deploy-preview-2749--oak-web-application.netlify.thenational.academy
2. Test some events are still working as expected

For devs:
1. pull down the branch and run on chrome
2. see that you can access feature flagged downloads-auth page
